test/powershell/engine/ResourceValidation/TestRunner.ps1

```diff
@@ -0,0 +1,57 @@
+function Test-ResourceStrings
+{
+    param ( $AssemblyName, $ExcludeList )
+
+    # determine the needed resource directory. If these tests are moved
+    # this logic will need to change
+    $repoBase = (Resolve-Path (Join-Path $psScriptRoot ../../../..)).Path
+    $asmBase = Join-Path $repoBase "src/$AssemblyName"
+    $resourceDir = Join-Path $asmBase resources
+    $resourceFiles = Get-ChildItem $resourceDir -Filter *.resx -ea stop | Where-Object {

```

Comment:
Why not use `-Exclude` parameter for `Get-ChildItem`?

Reply:
i can do that

Reply:
huh - it turns out that we might have a bug - `get-childitem -filter *.resx -exclude <thing>` returns no results. I'll have to leave it as is.

Reply:
Now that you mention it, there's a number of issues with `-exclude`.  Ok with leaving as-is